---
title: "RNA Transcription"
layout: page
---


### Learning Objectives

* Explain how RNA is synthesized using DNA as a template
* Distinguish between transcription in prokaryotes and eukaryotes

During the process of **transcription**{: data-type="term"}, the information encoded within the DNA sequence of one or more genes is transcribed into a strand of RNA, also called an **RNA transcript**{: data-type="term"}. The resulting single-stranded RNA molecule, composed of ribonucleotides containing the bases adenine (A), cytosine (C), guanine (G), and uracil (U), acts as a mobile molecular copy of the original DNA sequence. Transcription in prokaryotes and in eukaryotes requires the DNA double helix to partially unwind in the region of RNA synthesis. The unwound region is called a **transcription bubble**{: data-type="term"}. Transcription of a particular gene always proceeds from one of the two DNA strands that acts as a template, the so-called **antisense strand**{: data-type="term"}. The RNA product is complementary to the template strand of DNA and is almost identical to the nontemplate DNA strand, or the **sense strand**{: data-type="term"}. The only difference is that in RNA, all of the T nucleotides are replaced with U nucleotides; during RNA synthesis, U is incorporated when there is an A in the complementary antisense strand.

### Transcription in Bacteria

Bacteria use the same RNA polymerase to transcribe all of their genes. Like DNA polymerase, **RNA polymerase**{: data-type="term"} adds nucleotides one by one to the 3’-OH group of the growing nucleotide chain. One critical difference in activity between DNA polymerase and RNA polymerase is the requirement for a 3’-OH onto which to add nucleotides: DNA polymerase requires such a 3’-OH group, thus necessitating a primer, whereas RNA polymerase does not. During transcription, a ribonucleotide complementary to the DNA template strand is added to the growing RNA strand and a covalent phosphodiester bond is formed by dehydration synthesis between the new nucleotide and the last one added. In *E. coli*, RNA polymerase comprises six polypeptide subunits, five of which compose the polymerase core enzyme responsible for adding RNA nucleotides to a growing strand. The sixth subunit is known as sigma (σ). The **σ factor**{: data-type="term" .no-emphasis} enables RNA polymerase to bind to a specific promoter, thus allowing for the transcription of various genes. There are various σ factors that allow for transcription of various genes.

#### Initiation

The **initiation of transcription**{: data-type="term"} begins at a **promoter**{: data-type="term"}, a DNA sequence onto which the transcription machinery binds and initiates transcription. The nucleotide pair in the DNA double helix that corresponds to the site from which the first 5’ RNA nucleotide is transcribed is the initiation site. Nucleotides preceding the initiation site are designated “upstream,” whereas nucleotides following the initiation site are called “downstream” nucleotides. In most cases, promoters are located just upstream of the genes they regulate. <sup />Although promoter sequences vary among bacterial genomes, a few elements are conserved. At the –10 and –35 positions within the DNA prior to the initiation site (designated +1), there are two promoter consensus sequences, or regions that are similar across all promoters and across various bacterial species. The –10 consensus sequence, called the **TATA box**{: data-type="term" .no-emphasis}, is TATAAT. The –35 sequence is recognized and bound by σ.

#### Elongation

The **elongation in transcription**{: data-type="term"} phase begins when the σ subunit dissociates from the polymerase, allowing the core enzyme to synthesize RNA complementary to the DNA template in a 5’ to 3’ direction at a rate of approximately 40 nucleotides per second. As elongation proceeds, the DNA is continuously unwound ahead of the core enzyme and rewound behind it ([\[link\]](#OSC_Microbio_11_03_TxnElong)).

 ![Diagram of transcription. A double stranded piece of DNA has a large oval labeled RNA polymerase sitting on it just past a region labeled promoter. The DNA in the RNA polymerase has separated and the bottom DNA strand (labeled template strand) has a newly forming RNA strand attached to it. The RNA strand is being built from 5&#x2019; to 3&#x2019;. The other strand of DNA is the nontemplate strand and does not have RNA being built.](../resources/OSC_Microbio_11_03_TxnElong.jpg "During elongation, the bacterial RNA polymerase tracks along the DNA template, synthesizes mRNA in the 5&#x2019; to 3&#x2019; direction, and unwinds and rewinds the DNA as it is read."){: #OSC_Microbio_11_03_TxnElong}

#### Termination

Once a gene is transcribed, the bacterial polymerase must dissociate from the DNA template and liberate the newly made RNA. This is referred to as **termination of transcription**{: data-type="term"}. The DNA template includes repeated nucleotide sequences that act as termination signals, causing RNA polymerase to stall and release from the DNA template, freeing the RNA transcript.

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* Where does σ factor of RNA polymerase bind DNA to start transcription?
* What occurs to initiate the polymerization activity of RNA polymerase?
* Where does the signal to end transcription come from?
{: data-bullet-style="bullet"}

</div>

### Transcription in Eukaryotes

Prokaryotes and eukaryotes perform fundamentally the same process of transcription, with a few significant differences (see [\[link\]](#fs-id1167663533154)). Eukaryotes use three different polymerases, RNA polymerases I, II, and III, all structurally distinct from the bacterial **RNA polymerase**{: data-type="term" .no-emphasis}. Each transcribes a different subset of genes. Interestingly, **archaea**{: data-type="term" .no-emphasis} contain a single RNA polymerase that is more closely related to eukaryotic RNA polymerase II than to its bacterial counterpart. Eukaryotic mRNAs are also usually monocistronic, meaning that they each encode only a single polypeptide, whereas prokaryotic mRNAs of bacteria and archaea are commonly **polycistronic**{: data-type="term"}, meaning that they encode multiple polypeptides.

The most important difference between prokaryotes and eukaryotes is the latter’s membrane-bound nucleus, which influences the ease of use of RNA molecules for protein synthesis. With the genes bound in a nucleus, the eukaryotic cell must transport protein-encoding RNA molecules to the cytoplasm to be translated. Protein-encoding **primary transcript**{: data-type="term"}s, the RNA molecules directly synthesized by RNA polymerase, must undergo several processing steps to protect these RNA molecules from degradation during the time they are transferred from the nucleus to the cytoplasm and translated into a protein. For example, eukaryotic mRNAs may last for several hours, whereas the typical prokaryotic mRNA lasts no more than 5 seconds.

The **primary transcript**{: data-type="term" .no-emphasis} (also called pre-mRNA) is first coated with RNA-stabilizing proteins to protect it from degradation while it is processed and exported out of the nucleus. The first type of processing begins while the primary transcript is still being synthesized; a special 7-methylguanosine nucleotide, called the **5’ cap**{: data-type="term"}, is added to the 5’ end of the growing transcript. In addition to preventing degradation, factors involved in subsequent protein synthesis recognize the cap, which helps initiate translation by ribosomes. Once elongation is complete, another processing enzyme then adds a string of approximately 200 adenine nucleotides to the 3’ end, called the **poly-A tail**{: data-type="term"}. This modification further protects the pre-mRNA from degradation and signals to cellular factors that the transcript needs to be exported to the cytoplasm.

Eukaryotic genes that encode polypeptides are composed of coding sequences called **exon**{: data-type="term"}s (*ex*-on signifies that they are *ex*pressed) and intervening sequences called **intron**{: data-type="term"}s (*int*-ron denotes their *int*ervening role). Transcribed RNA sequences corresponding to introns do not encode regions of the functional polypeptide and are removed from the pre-mRNA during processing. It is essential that all of the intron-encoded RNA sequences are completely and precisely removed from a pre-mRNA before protein synthesis so that the exon-encoded RNA sequences are properly joined together to code for a functional polypeptide. If the process errs by even a single nucleotide, the sequences of the rejoined exons would be shifted, and the resulting polypeptide would be nonfunctional. The process of removing intron-encoded RNA sequences and reconnecting those encoded by exons is called **RNA splicing**{: data-type="term"} and is facilitated by the action of a **spliceosome**{: data-type="term"} containing small nuclear ribonucleo proteins (snRNPs). Intron-encoded RNA sequences are removed from the pre-mRNA while it is still in the nucleus. Although they are not translated, introns appear to have various functions, including gene regulation and mRNA transport. On completion of these modifications, the **mature transcript**{: data-type="term" .no-emphasis}, the mRNA that encodes a polypeptide, is transported out of the nucleus, destined for the cytoplasm for translation. Introns can be spliced out differently, resulting in various exons being included or excluded from the final mRNA product. This process is known as **alternative splicing**{: data-type="term" .no-emphasis}. The advantage of alternative splicing is that different types of mRNA transcripts can be generated, all derived from the same DNA sequence. In recent years, it has been shown that some archaea also have the ability to splice their pre-mRNA.

<table summary="Table titled: Comparison of Transcription in Bacteria Versus Eukaryote. For number of polypeptides encoded per mRNA &#x2013; bacteria are monocistronic or polycistronic while eukaryotes are exclusively monocistronic. In bacteria the holoenzyme (core + sigma) is responsible for strand elongation. In eukaryotes RNA polymerase I, II, or III are responsible for strand elongation. Eukaryotes have the addition of a 5&#x2019; cap, bacteria do not. Eukaryotes have the addition of a 3&#x2019; poly-A tail, bacteria do not. Eukaryotes have splicing of pre-mRNA, bacteria do not." class="span-all"><thead>
<tr>
<th colspan="3" data-align="center">Comparison of Transcription in Bacteria Versus Eukaryotes</th>
</tr>
<tr valign="top">
<th data-valign="top" data-align="center">Property</th>
<th data-valign="top" data-align="center">Bacteria</th>
<th data-valign="top" data-align="center">Eukaryotes</th>
</tr>
</thead><tbody>
<tr valign="top">
<td data-valign="top" data-align="left">Number of polypeptides encoded per mRNA</td>
<td data-valign="top" data-align="left">Monocistronic or polycistronic</td>
<td data-valign="top" data-align="left">Exclusively monocistronic</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">Strand elongation</td>
<td data-valign="top" data-align="left">core + σ = holoenzyme</td>
<td data-valign="top" data-align="left">RNA polymerases I, II, or III</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">Addition of 5’ cap</td>
<td data-valign="top" data-align="left">No</td>
<td data-valign="top" data-align="left">Yes</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">Addition of 3’ poly-A tail</td>
<td data-valign="top" data-align="left">No</td>
<td data-valign="top" data-align="left">Yes</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">Splicing of pre-mRNA</td>
<td data-valign="top" data-align="left">No</td>
<td data-valign="top" data-align="left">Yes</td>
</tr>
</tbody></table>

<div data-type="note" class="microbiology link-to-learning" markdown="1">
<span data-type="media" data-alt="Link to learning graphic"> ![Link to learning graphic](../resources/OSC_Interactive_200.png) </span>
Visualize how [mRNA splicing][1] happens by watching the process in action in this video. See how introns are removed during [RNA splicing][2] here.

</div>

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* In eukaryotic cells, how is the RNA transcript from a gene for a protein modified after it is transcribed?
* Do exons or introns contain information for protein sequences?
{: data-bullet-style="bullet"}

</div>

<div data-type="note" class="microbiology clinical-focus" markdown="1">
<div data-type="title">
Part 2
</div>
In the emergency department, a nurse told Mark that he had made a good decision to come to the hospital because his symptoms indicated an infection that had gotten out of control. Mark’s symptoms had progressed, with the area of skin affected and the amount of swelling increasing. Within the affected area, a rash had begun, blistering and small gas pockets underneath the outermost layer of skin had formed, and some of the skin was becoming gray. Based on the putrid smell of the pus draining from one of the blisters, the rapid progression of the infection, and the visual appearance of the affected skin, the physician immediately began treatment for necrotizing fasciitis. Mark’s physician ordered a culture of the fluid draining from the blister and also ordered blood work, including a white blood cell count.

Mark was admitted to the intensive care unit and began intravenous administration of a broad-spectrum antibiotic to try to minimize further spread of the infection. Despite antibiotic therapy, Mark’s condition deteriorated quickly. Mark became confused and dizzy. Within a few hours of his hospital admission, his blood pressure dropped significantly and his breathing became shallower and more rapid. Additionally, blistering increased, with the blisters intensifying in color to purplish black, and the wound itself seemed to be progressing rapidly up Mark’s leg.

* What are possible causative agents of Mark’s necrotizing fasciitis?
* What are some possible explanations for why the antibiotic treatment does not seem to be working?
{: data-bullet-style="bullet"}

*Jump to the [next](/m58845#fs-id1167661565061) Clinical Focus box. Go back to the [previous](/m58840#fs-id1167662419529) Clinical Focus box.*

</div>

### Key Concepts and Summary

* During **transcription**, the information encoded in DNA is used to make RNA.
* **RNA polymerase** synthesizes RNA, using the antisense strand of the DNA as template by adding complementary RNA nucleotides to the 3’ end of the growing strand.
* RNA polymerase binds to DNA at a sequence called a **promoter** during the **initiation of transcription**.
* Genes encoding proteins of related functions are frequently transcribed under the control of a single promoter in prokaryotes, resulting in the formation of a **polycistronic mRNA** molecule that encodes multiple polypeptides.
* Unlike DNA polymerase, RNA polymerase does not require a 3’-OH group to add nucleotides, so a **primer** is not needed during initiation.
* **Termination of transcription** in bacteria occurs when the RNA polymerase encounters specific DNA sequences that lead to stalling of the polymerase. This results in release of RNA polymerase from the DNA template strand, freeing the **RNA transcript**.
* Eukaryotes have three different RNA polymerases. Eukaryotes also have monocistronic mRNA, each encoding only a single polypeptide.
* Eukaryotic primary transcripts are processed in several ways, including the addition of a **5’ cap** and a 3′-**poly-A tail**, as well as **splicing**, to generate a mature mRNA molecule that can be transported out of the nucleus and that is protected from degradation.
{: data-bullet-style="bullet"}

### Multiple Choice

<div data-type="exercise">
<div data-type="problem" markdown="1">
During which stage of bacterial transcription is the σ subunit of the RNA polymerase involved?

1.  initiation
2.  elongation
3.  termination
4.  splicing
{: type="A"}

</div>
<div data-type="solution" markdown="1">
A

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Which of the following components is involved in the initiation of transcription?

1.  primer
2.  origin
3.  promoter
4.  start codon
{: type="A"}

</div>
<div data-type="solution" markdown="1">
C

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Which of the following is not a function of the 5’ cap and 3’ poly-A tail of a mature eukaryotic mRNA molecule?

1.  to facilitate splicing
2.  to prevent mRNA degradation
3.  to aid export of the mature transcript to the cytoplasm
4.  to aid ribosome binding to the transcript
{: type="A"}

</div>
<div data-type="solution" markdown="1">
A

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Mature mRNA from a eukaryote would contain each of these features except which of the following?

1.  exon-encoded RNA
2.  intron-encoded RNA
3.  5’ cap
4.  3’ poly-A tail
{: type="A"}

</div>
<div data-type="solution" markdown="1">
B

</div>
</div>

### Fill in the Blank

<div data-type="exercise">
<div data-type="problem" markdown="1">
A \_\_\_\_\_\_\_\_ mRNA is one that codes for multiple polypeptides.

</div>
<div data-type="solution" markdown="1">
polycistronic

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
The protein complex responsible for removing intron-encoded RNA sequences from primary transcripts in eukaryotes is called the \_\_\_\_\_\_\_\_.

</div>
<div data-type="solution" markdown="1">
Spliceosome

</div>
</div>

### Short Answer

<div data-type="exercise">
<div data-type="problem" markdown="1">
What is the purpose of RNA processing in eukaryotes? Why don’t prokaryotes require similar processing?

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Below is a DNA sequence. Envision that this is a section of a DNA molecule that has separated in preparation for transcription, so you are only seeing the antisense strand. Construct the mRNA sequence transcribed from this template.

Antisense DNA strand: 3’-T A C T G A C T G A C G A T C-5’

</div>
</div>

### Critical Thinking

<div data-type="exercise">
<div data-type="problem" markdown="1">
Predict the effect of an alteration in the sequence of nucleotides in the –35 region of a bacterial promoter.

</div>
</div>



[1]: https://openstax.org/l/22mrnasplice
[2]: https://openstax.org/l/22rnasplice
