---
title: "Mutations"
layout: page
---


### Learning Objectives

* Compare point mutations and frameshift mutations
* Describe the differences between missense, nonsense, and silent mutations
* Describe the differences between light and dark repair
* Explain how different mutagens act
* Explain why the Ames test can be used to detect carcinogens
* Analyze sequences of DNA and identify examples of types of mutations

A **mutation**{: data-type="term"} is a heritable change in the DNA sequence of an organism. The resulting organism, called a **mutant**{: data-type="term"}, may have a recognizable change in **phenotype**{: data-type="term" .no-emphasis} compared to the **wild type**{: data-type="term"}, which is the phenotype most commonly observed in nature. A change in the DNA sequence is conferred to mRNA through transcription, and may lead to an altered amino acid sequence in a protein on translation. Because proteins carry out the vast majority of cellular functions, a change in amino acid sequence in a protein may lead to an altered phenotype for the cell and organism.

### Effects of Mutations on DNA Sequence

There are several types of mutations that are classified according to how the DNA molecule is altered. One type, called a **point mutation**{: data-type="term"}, affects a single base and most commonly occurs when one base is substituted or replaced by another. Mutations also result from the addition of one or more bases, known as an **insertion**{: data-type="term"}, or the removal of one or more bases, known as a **deletion**{: data-type="term"}.

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* What type of a mutation occurs when a gene has two fewer nucleotides in its sequence?
{: data-bullet-style="bullet"}

</div>

### Effects of Mutations on Protein Structure and Function

Point mutations may have a wide range of effects on protein function ([\[link\]](#OSC_Microbio_11_05_EffMut)). As a consequence of the **degeneracy**{: data-type="term" .no-emphasis} of the genetic code, a point mutation will commonly result in the same amino acid being incorporated into the resulting polypeptide despite the sequence change. This change would have no effect on the protein’s structure, and is thus called a **silent mutation**{: data-type="term"}. A **missense mutation**{: data-type="term"} results in a different amino acid being incorporated into the resulting polypeptide. The effect of a missense mutation depends on how chemically different the new amino acid is from the wild-type amino acid. The location of the changed amino acid within the protein also is important. For example, if the changed amino acid is part of the enzyme’s **active site**{: data-type="term" .no-emphasis}, then the effect of the missense mutation may be significant. Many missense mutations result in proteins that are still functional, at least to some degree. Sometimes the effects of missense mutations may be only apparent under certain environmental conditions; such missense mutations are called **conditional mutation**{: data-type="term"}s. Rarely, a missense mutation may be beneficial. Under the right environmental conditions, this type of mutation may give the organism that harbors it a selective advantage. Yet another type of point mutation, called a **nonsense mutation**{: data-type="term"}, converts a codon encoding an amino acid (a sense codon) into a stop codon (a nonsense codon). Nonsense mutations result in the synthesis of proteins that are shorter than the wild type and typically not functional.

Deletions and insertions also cause various effects. Because codons are triplets of nucleotides, insertions or deletions in groups of three nucleotides may lead to the insertion or deletion of one or more amino acids and may not cause significant effects on the resulting protein’s functionality. However, **frameshift mutation**{: data-type="term"}s, caused by insertions or deletions of a number of nucleotides that are not a multiple of three are extremely problematic because a shift in the reading frame results ([\[link\]](#OSC_Microbio_11_05_EffMut)). Because ribosomes read the mRNA in triplet codons, frameshift mutations can change every amino acid after the point of the mutation. The new reading frame may also include a stop codon before the end of the coding sequence. Consequently, proteins made from genes containing frameshift mutations are nearly always nonfunctional.

 ![Diagram of point mutations: substitution of a single base. A silent mutation has no effect on the protein sequence. The diagram shows a single letter change in the DNA which produces an RNA with a single letter that is different, however the protein is still the same. Missense results in an amino acid substitution. In this example the DNA has a single letter that has changed, which produces a single letter that is different on the mRNA, which produces a single amino acid that is different in the polypeptide chain. Nonsense mutations occur when a stop codon is substituted for an amino acid. The DNA has a single letter change, which changes one amino acid the mRNA, which produces a stop codon where there was an amino acid. Frameshift mutations are insertions or deletions of one or more bases which results in a reading frame shift. Two letters are deleted in the DNA strand which produces a shorter RNA strand which produces an entirely different protein.](../resources/OSC_Microbio_11_05_EffMut.jpg "Mutations can lead to changes in the protein sequence encoded by the DNA."){: #OSC_Microbio_11_05_EffMut}

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* What are the reasons a nucleotide change in a gene for a protein might not have any effect on the phenotype of that gene?
* Is it possible for an insertion of three nucleotides together after the fifth nucleotide in a protein-coding gene to produce a protein that is shorter than normal? How or how not?
{: data-bullet-style="bullet"}

</div>

<div data-type="note" class="microbiology micro-connection" markdown="1">
<div data-type="title">
A Beneficial Mutation
</div>
Since the first case of infection with human immunodeficiency virus (HIV) was reported in 1981, nearly 40 million people have died from HIV infection,<sup data-type="footnote-number" id="footnote-ref1">[1](#footnote1){: data-type="footnote-link"}</sup> the virus that causes acquired immune deficiency syndrome (AIDS). The virus targets helper T cells that play a key role in bridging the innate and adaptive immune response, infecting and killing cells normally involved in the body’s response to infection. There is no cure for HIV infection, but many drugs have been developed to slow or block the progression of the virus. Although individuals around the world may be infected, the highest prevalence among people 15–49 years old is in sub-Saharan Africa, where nearly one person in 20 is infected, accounting for greater than 70% of the infections worldwide<sup data-type="footnote-number" id="footnote-ref2">[2](#footnote2){: data-type="footnote-link"}</sup> ([\[link\]](#OSC_Microbio_11_05_HIVPrev)). Unfortunately, this is also a part of the world where prevention strategies and drugs to treat the infection are the most lacking.

In recent years, scientific interest has been piqued by the discovery of a few individuals from northern Europe who are resistant to HIV infection. In 1998, American geneticist Stephen J. O’Brien at the National Institutes of Health (NIH) and colleagues published the results of their genetic analysis of more than 4,000 individuals. These indicated that many individuals of Eurasian descent (up to 14% in some ethnic groups) have a deletion mutation, called CCR5-delta 32, in the gene encoding CCR5. CCR5 is a coreceptor found on the surface of T cells that is necessary for many strains of the virus to enter the host cell. The mutation leads to the production of a receptor to which HIV cannot effectively bind and thus blocks viral entry. People homozygous for this mutation have greatly reduced susceptibility to HIV infection, and those who are heterozygous have some protection from infection as well.

It is not clear why people of northern European descent, specifically, carry this mutation, but its prevalence seems to be highest in northern Europe and steadily decreases in populations as one moves south. Research indicates that the mutation has been present since before HIV appeared and may have been selected for in European populations as a result of exposure to the plague or smallpox. This mutation may protect individuals from plague (caused by the bacterium *Yersinia pestis*) and smallpox (caused by the variola virus) because this receptor may also be involved in these diseases. The age of this mutation is a matter of debate, but estimates suggest it appeared between 1875 years to 225 years ago, and may have been spread from Northern Europe through Viking invasions.

This exciting finding has led to new avenues in HIV research, including looking for drugs to block CCR5 binding to HIV in individuals who lack the mutation. Although DNA testing to determine which individuals carry the CCR5-delta 32 mutation is possible, there are documented cases of individuals homozygous for the mutation contracting HIV. For this reason, DNA testing for the mutation is not widely recommended by public health officials so as not to encourage risky behavior in those who carry the mutation. Nevertheless, inhibiting the binding of HIV to CCR5 continues to be a valid strategy for the development of drug therapies for those infected with HIV.

![Map of global prevalence of HIV in 2015. Global rate is 0.8%. Middle East and North Africa = 0.1%. Asia and the Pacific = 0.2%. Western and Central Europe and North America = 0.3%. Latin America and the Caribbean = 0.5%. Eastern Europe and Central Asia = 0.9%. West and Central Africa = 2.2%. East and Southern Africa = 7.1%](../resources/OSC_Microbio_11_05_HIVPrev.jpg "HIV is highly prevalent in sub-Saharan Africa, but its prevalence is quite low in some other parts of the world."){: #OSC_Microbio_11_05_HIVPrev}


</div>

### Causes of Mutations

Mistakes in the process of DNA replication can cause **spontaneous mutation**{: data-type="term"}**s** to occur. The error rate of **DNA polymerase**{: data-type="term" .no-emphasis} is one incorrect base per billion base pairs replicated. Exposure to **mutagen**{: data-type="term"}**s** can cause **induced mutation**{: data-type="term"}**s**, which are various types of chemical agents or radiation ([\[link\]](#fs-id1167662401623)). Exposure to a mutagen can increase the rate of mutation more than 1000-fold. Mutagens are often also **carcinogen**{: data-type="term"}**s**, agents that cause cancer. However, whereas nearly all carcinogens are mutagenic, not all mutagens are necessarily carcinogens.

#### Chemical Mutagens

Various types of chemical mutagens interact directly with DNA either by acting as nucleoside analogs or by modifying nucleotide bases. Chemicals called **nucleoside analog**{: data-type="term"}**s** are structurally similar to normal nucleotide bases and can be incorporated into DNA during replication ([\[link\]](#OSC_Microbio_11_05_NucAnMod)). These base analogs induce mutations because they often have different base-pairing rules than the bases they replace. Other chemical mutagens can modify normal DNA bases, resulting in different base-pairing rules. For example, nitrous acid deaminates cytosine, converting it to uracil. Uracil then pairs with adenine in a subsequent round of replication, resulting in the conversion of a GC base pair to an AT base pair. Nitrous acid also deaminates adenine to hypoxanthine, which base pairs with cytosine instead of thymine, resulting in the conversion of a TA base pair to a CG base pair.

Chemical mutagens known as **intercalating agent**{: data-type="term"}**s** work differently. These molecules slide between the stacked nitrogenous bases of the DNA double helix, distorting the molecule and creating atypical spacing between nucleotide base pairs ([\[link\]](#OSC_Microbio_11_05_Intercal)). As a result, during DNA replication, DNA polymerase may either skip replicating several nucleotides (creating a **deletion**{: data-type="term" .no-emphasis}) or insert extra nucleotides (creating an **insertion**{: data-type="term" .no-emphasis}). Either outcome may lead to a **frameshift mutation**{: data-type="term" .no-emphasis}. Combustion products like polycyclic aromatic hydrocarbons are particularly dangerous intercalating agents that can lead to mutation-caused cancers. The intercalating agents **ethidium bromide**{: data-type="term" .no-emphasis} and **acridine orange**{: data-type="term" .no-emphasis} are commonly used in the laboratory to stain DNA for visualization and are potential mutagens.

 ![Diagram showing analogs of normal nitrogenous bases. Adenine nucleoside has a double ring of carbon and nitrogen with an NH2 group attached at one of the carbons. The analog 2-aminopurine nucleoside has an H attached to this carbon. Thymine nucleoside has a single carbon nitrogen ring with a CH3 attached to the carbon at the bottom of the ring. The analog 5-bromouracil nucleoside has a Br attached to this carbon. Cytosine has a single carbon and nitrogen ring with an NH2 at one of the carbons. Nitrous acid (NHO2) replaces the NH2 with a double bonded O. This converts the cytosine to a uracil with now binds with adenine instead of guanine.](../resources/OSC_Microbio_11_05_NucAnMod.jpg "(a) 2-aminopurine nucleoside (2AP) structurally is a nucleoside analog to adenine nucleoside, whereas 5-bromouracil (5BU) is a nucleoside analog to thymine nucleoside. 2AP base pairs with C, converting an AT base pair to a GC base pair after several rounds of replication. 5BU pairs with G, converting an AT base pair to a GC base pair after several rounds of replication. (b) Nitrous acid is a different type of chemical mutagen that modifies already existing nucleoside bases like C to produce U, which base pairs with A. This chemical modification, as shown here, results in converting a CG base pair to a TA base pair."){: #OSC_Microbio_11_05_NucAnMod}

![Acridine (a molecule with 3 rings and a nitrogen group at either end binds between the two strands of a normal parent DNA. When this is replicated nucleotides can either be deleted or added to produce DNA that is either shorter or longer than the original parent molecule.](../resources/OSC_Microbio_11_05_Intercal.jpg "Intercalating agents, such as acridine, introduce atypical spacing between base pairs, resulting in DNA polymerase introducing either a deletion or an insertion, leading to a potential frameshift mutation."){: #OSC_Microbio_11_05_Intercal}

#### Radiation

Exposure to either ionizing or nonionizing radiation can each induce mutations in DNA, although by different mechanisms. Strong **ionizing radiation**{: data-type="term"} like **X-rays**{: data-type="term" .no-emphasis} and **gamma rays**{: data-type="term" .no-emphasis} can cause single- and double-stranded breaks in the DNA backbone through the formation of hydroxyl radicals on radiation exposure ([\[link\]](#OSC_Microbio_11_05_Rad)). Ionizing radiation can also modify bases; for example, the deamination of cytosine to uracil, analogous to the action of nitrous acid.<sup data-type="footnote-number" id="footnote-ref3">[3](#footnote3){: data-type="footnote-link"}</sup> Ionizing radiation exposure is used to kill microbes to sterilize medical devices and foods, because of its dramatic nonspecific effect in damaging DNA, proteins, and other cellular components (see [Using Physical Methods to Control Microorganisms](/m58854){: .target-chapter}).

Nonionizing radiation, like ultraviolet light, is not energetic enough to initiate these types of chemical changes. However, **nonionizing radiation**{: data-type="term"} can induce dimer formation between two adjacent pyrimidine bases, commonly two thymines, within a nucleotide strand. During **thymine dimer**{: data-type="term"} formation, the two adjacent thymines become covalently linked and, if left unrepaired, both DNA replication and transcription are stalled at this point. DNA polymerase may proceed and replicate the dimer incorrectly, potentially leading to frameshift or point mutations.

 ![a) Ionizing radiation (such as X-rays or gamma-rays) create double stranded breaks in DNA (breaks in the backbone). B) Non-ionizing radiation (such as ultraviolet light) causes Ts on the same strand of DNA to bind to each other rather than to the As across from them. This causes a kink in the DNA strand.](../resources/OSC_Microbio_11_05_Rad.jpg "(a) Ionizing radiation may lead to the formation of single-stranded and double-stranded breaks in the sugar-phosphate backbone of DNA, as well as to the modification of bases (not shown). (b) Nonionizing radiation like ultraviolet light can lead to the formation of thymine dimers, which can stall replication and transcription and introduce frameshift or point mutations."){: #OSC_Microbio_11_05_Rad}

<table summary="Table titled: A Summary of Mutagenic Agents. Columns: Mutagenic Agents; Mode of Action; Effect on DNA; Resulting Type of Mutation. First grouping is Nucleoside analogs. Mutagenic Agent: 2-aminopurine; Is inserted in place of A but base pairs with C;l converts AT to GC base pair; Point. Mutagenic Agent: 5-bromouracil ; Is inserted in place of T but base pairs with G; Converts AT to GC base pair; Point. Next groupin is Nucleotide-modifying agent. Mutagenic Agent: Nitrous oxide; Deaminates C to U; Converts GC to AT base pair; Point. Next group is: Intercalating agents. Mutagenic Agent: Acridine; orange, ethidium bromide, polycyclic aromatic hydrocarbons; Distorts double helix, creates unusual spacing between nucleotides; Introduces small deletions and insertions; Frameshift. Next group is Ionizing radiation. Mutagenic Agent: X-rays, &#x3B3;-rays; Forms hydroxyl radicals; Causes single- and double-strand DNA breaks; Repair mechanisms may introduce mutations. Mutagenic Agent: X-rays, &#x3B3;-rays; Modifies bases (e.g., deaminating C to U); Converts GC to AT base pair; Point. Next group is: Nonionizing radiation. Mutagenic Agent: Ultraviolet; Forms pyrimidine (usually thymine) dimers; Causes DNA replication errors; Frameshift or point." class="span-all"><thead>
<tr>
<th colspan="4" data-align="center">A Summary of Mutagenic Agents</th>
</tr>
<tr valign="top">
<th data-valign="top" data-align="left">Mutagenic Agents</th>
<th data-valign="top" data-align="left">Mode of Action</th>
<th data-valign="top" data-align="left">Effect on DNA</th>
<th data-valign="top" data-align="left">Resulting Type of Mutation</th>
</tr>
</thead><tbody>
<tr valign="top">
<td colspan="4" data-valign="top" data-align="center">Nucleoside analogs</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">2-aminopurine</td>
<td data-valign="top" data-align="left">Is inserted in place of A but base pairs with C</td>
<td data-valign="top" data-align="left">Converts AT to GC base pair</td>
<td data-valign="top" data-align="left">Point</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">5-bromouracil</td>
<td data-valign="top" data-align="left">Is inserted in place of T but base pairs with G</td>
<td data-valign="top" data-align="left">Converts AT to GC base pair</td>
<td data-valign="top" data-align="left">Point</td>
</tr>
<tr valign="top">
<td colspan="4" data-valign="top" data-align="left">Nucleotide-modifying agent</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">Nitrous oxide</td>
<td data-valign="top" data-align="left">Deaminates C to U</td>
<td data-valign="top" data-align="left">Converts GC to AT base pair</td>
<td data-valign="top" data-align="left">Point</td>
</tr>
<tr valign="top">
<td colspan="4" data-valign="top" data-align="center">Intercalating agents</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">Acridine orange, ethidium bromide, polycyclic aromatic hydrocarbons</td>
<td data-valign="top" data-align="left">Distorts double helix, creates unusual spacing between nucleotides</td>
<td data-valign="top" data-align="left">Introduces small deletions and insertions</td>
<td data-valign="top" data-align="left">Frameshift</td>
</tr>
<tr valign="top">
<td colspan="4" data-valign="top" data-align="left">Ionizing radiation</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">X-rays, γ-rays</td>
<td data-valign="top" data-align="left">Forms hydroxyl radicals</td>
<td data-valign="top" data-align="left">Causes single- and double-strand DNA breaks</td>
<td data-valign="top" data-align="left">Repair mechanisms may introduce mutations</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">X-rays, γ-rays</td>
<td data-valign="top" data-align="left">Modifies bases (e.g., deaminating C to U)</td>
<td data-valign="top" data-align="left">Converts GC to AT base pair</td>
<td data-valign="top" data-align="left">Point</td>
</tr>
<tr valign="top">
<td colspan="4" data-valign="top" data-align="center">Nonionizing radiation</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left">Ultraviolet</td>
<td data-valign="top" data-align="left">Forms pyrimidine (usually thymine) dimers</td>
<td data-valign="top" data-align="left">Causes DNA replication errors</td>
<td data-valign="top" data-align="left">Frameshift or point</td>
</tr>
</tbody></table>

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* How does a base analog introduce a mutation?
* How does an intercalating agent introduce a mutation?
* What type of mutagen causes thymine dimers?
{: data-bullet-style="bullet"}

</div>

### DNA Repair

The process of **DNA replication**{: data-type="term" .no-emphasis} is highly accurate, but mistakes can occur spontaneously or be induced by mutagens. Uncorrected mistakes can lead to serious consequences for the phenotype. Cells have developed several repair mechanisms to minimize the number of mutations that persist.

#### Proofreading

Most of the mistakes introduced during DNA replication are promptly corrected by most **DNA polymerase**{: data-type="term" .no-emphasis}s through a function called proofreading. In **proofreading**{: data-type="term" .no-emphasis}, the DNA polymerase reads the newly added base, ensuring that it is complementary to the corresponding base in the template strand before adding the next one. If an incorrect base has been added, the enzyme makes a cut to release the wrong nucleotide and a new base is added.

#### Mismatch Repair

Some errors introduced during replication are corrected shortly after the replication machinery has moved. This mechanism is called **mismatch repair**{: data-type="term" .no-emphasis}. The enzymes involved in this mechanism recognize the incorrectly added nucleotide, excise it, and replace it with the correct base. One example is the methyl-directed mismatch repair in ***E. coli***{: data-type="term" .no-emphasis}. The DNA is hemimethylated. This means that the parental strand is methylated while the newly synthesized daughter strand is not. It takes several minutes before the new strand is methylated. Proteins MutS, MutL, and MutH bind to the hemimethylated site where the incorrect nucleotide is found. MutH cuts the nonmethylated strand (the new strand). An exonuclease removes a portion of the strand (including the incorrect nucleotide). The gap formed is then filled in by DNA pol III and ligase.

#### Repair of Thymine Dimers

Because the production of **thymine dimer**{: data-type="term" .no-emphasis}s is common (many organisms cannot avoid ultraviolet light), mechanisms have evolved to repair these lesions. In **nucleotide excision repair**{: data-type="term"} (also called dark repair), enzymes remove the pyrimidine dimer and replace it with the correct nucleotides ([\[link\]](#OSC_Microbio_11_05_ThyDiRep)). In *E. coli*, the DNA is scanned by an enzyme complex. If a distortion in the double helix is found that was introduced by the pyrimidine dimer, the enzyme complex cuts the sugar-phosphate backbone several bases upstream and downstream of the dimer, and the segment of DNA between these two cuts is then enzymatically removed. DNA pol I replaces the missing nucleotides with the correct ones and DNA ligase seals the gap in the sugar-phosphate backbone.

The **direct repair**{: data-type="term"} (also called light repair) of thymine dimers occurs through the process of **photoreactivation**{: data-type="term"} in the presence of visible light. An enzyme called **photolyase**{: data-type="term" .no-emphasis} recognizes the distortion in the DNA helix caused by the thymine dimer and binds to the dimer. Then, in the presence of visible light, the photolyase enzyme changes conformation and breaks apart the thymine dimer, allowing the thymines to again correctly base pair with the adenines on the complementary strand. Photoreactivation appears to be present in all organisms, with the exception of placental mammals, including humans. Photoreactivation is particularly important for organisms chronically exposed to **ultraviolet radiation**{: data-type="term" .no-emphasis}, like plants, photosynthetic bacteria, algae, and corals, to prevent the accumulation of mutations caused by thymine dimer formation.

 ![a) Diagram of nucleotide excision repair. A thymine dimer is when 2 Ts are bound to each other rather than to the As across from them. This damage is recognized and a large section of DNA on either side of the dimer is removed. Nuclease is used to cut the strands and helicase removes the damaged section. DNA pol I and DNA ligase make the repairs. B) in photoreactivation, an enzyme called photolyase sits binds to the thymine dimer and breaks the bonds between the Ts. The Ts can then bind to the As across from them.](../resources/OSC_Microbio_11_05_ThyDiRep.jpg "Bacteria have two mechanisms for repairing thymine dimers. (a) In nucleotide excision repair, an enzyme complex recognizes the distortion in the DNA complex around the thymine dimer and cuts and removes the damaged DNA strand. The correct nucleotides are replaced by DNA pol I and the nucleotide strand is sealed by DNA ligase. (b) In photoreactivation, the enzyme photolyase binds to the thymine dimer and, in the presence of visible light, breaks apart the dimer, restoring the base pairing of the thymines with complementary adenines on the opposite DNA strand."){: #OSC_Microbio_11_05_ThyDiRep}

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* During mismatch repair, how does the enzyme recognize which is the new and which is the old strand?
* How does an intercalating agent introduce a mutation?
* What type of mutation does photolyase repair?
{: data-bullet-style="bullet"}

</div>

### Identifying Bacterial Mutants

One common technique used to identify bacterial mutants is called **replica plating**{: data-type="term"}. This technique is used to detect nutritional mutants, called **auxotroph**{: data-type="term"}s, which have a mutation in a gene encoding an enzyme in the biosynthesis pathway of a specific nutrient, such as an amino acid. As a result, whereas wild-type cells retain the ability to grow normally on a medium lacking the specific nutrient, auxotrophs are unable to grow on such a medium. During replica plating ([\[link\]](#OSC_Microbio_11_05_PosNegSel)), a population of bacterial cells is mutagenized and then plated as individual cells on a complex nutritionally complete plate and allowed to grow into colonies. Cells from these colonies are removed from this master plate, often using sterile velvet. This velvet, containing cells, is then pressed in the same orientation onto plates of various media. At least one plate should also be nutritionally complete to ensure that cells are being properly transferred between the plates. The other plates lack specific nutrients, allowing the researcher to discover various auxotrophic mutants unable to produce specific nutrients. Cells from the corresponding colony on the nutritionally complete plate can be used to recover the mutant for further study.

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* Why are cells plated on a nutritionally complete plate in addition to nutrient-deficient plates when looking for a mutant?
{: data-bullet-style="bullet"}

</div>

### The Ames Test

The **Ames test**{: data-type="term"}, developed by Bruce **Ames**{: data-type="term" .no-emphasis} (1928–) in the 1970s, is a method that uses bacteria for rapid, inexpensive screening of the carcinogenic potential of new chemical compounds. The test measures the mutation rate associated with exposure to the compound, which, if elevated, may indicate that exposure to this compound is associated with greater cancer risk. The Ames test uses as the test organism a strain of ***Salmonella typhimurium***{: data-type="term" .no-emphasis} that is a histidine auxotroph, unable to synthesize its own histidine because of a mutation in an essential gene required for its synthesis. After exposure to a potential mutagen, these bacteria are plated onto a medium lacking histidine, and the number of mutants regaining the ability to synthesize histidine is recorded and compared with the number of such mutants that arise in the absence of the potential mutagen ([\[link\]](#OSC_Microbio_11_05_Ames)). Chemicals that are more mutagenic will bring about more mutants with restored histidine synthesis in the Ames test. Because many chemicals are not directly mutagenic but are metabolized to mutagenic forms by liver enzymes, rat liver extract is commonly included at the start of this experiment to mimic liver metabolism. After the Ames test is conducted, compounds identified as mutagenic are further tested for their potential carcinogenic properties by using other models, including animal models like mice and rats.

 ![Diagram of the process of identifying auxotrophic mutants. First a medium containing histidine grows colonies of bacteria. Then, a sterile velvet surface is pressed onto the plate to pick up cells from bacterial colonies. Next the cells on the velvet are transferred to new plates, one with histidine and one without. A mark on the plate ensures that the orientation of the colonies is the same for all plates. After the plates are incubated, compare the growth on the plates to identify auxotrophic mutants that grow on medium containing histidine but do not grow on medium lacking histidine. A colony that is missing on the medium lacking histamine but present on the medium with histamine is the auxotrophic mutant.](../resources/OSC_Microbio_11_05_PosNegSel.jpg "Identification of auxotrophic mutants, like histidine auxotrophs, is done using replica plating. After mutagenesis, colonies that grow on nutritionally complete medium but not on medium lacking histidine are identified as histidine auxotrophs."){: #OSC_Microbio_11_05_PosNegSel}

![Diagram of the Ames test. 1 &#x2013; Add rat liver extract and Salmonella to control tube. Add rat liver extract, possible mutagen, and Salmonella to experimental tube. The Salmonella strain in this test requires histidine. 2 &#x2013; Plant and incubate both samples using medium lacking histidine. 3 &#x2013; Compare growth on plates to identify revertants, which suggest mutagen causes mutations. In the image the plate without the possible mutagen has a few colonies (control with natural revertants). The plate from the sample with the possible mutagen has many colonies (high number of revertants his- to his+).](../resources/OSC_Microbio_11_05_Ames.jpg "The Ames test is used to identify mutagenic, potentially carcinogenic chemicals. A Salmonella histidine auxotroph is used as the test strain, exposed to a potential mutagen/carcinogen. The number of reversion mutants capable of growing in the absence of supplied histidine is counted and compared with the number of natural reversion mutants that arise in the absence of the potential mutagen."){: #OSC_Microbio_11_05_Ames}

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* What mutation is used as an indicator of mutation rate in the Ames test?
* Why can the Ames test work as a test for carcinogenicity?
{: data-bullet-style="bullet"}

</div>

### Key Concepts and Summary

* A **mutation** is a heritable change in DNA. A mutation may lead to a change in the amino-acid sequence of a protein, possibly affecting its function.
* A **point mutation** affects a single base pair. A point mutation may cause a **silent mutation** if the mRNA codon codes for the same amino acid, a **missense mutation** if the mRNA codon codes for a different amino acid, or a **nonsense mutation** if the mRNA codon becomes a stop codon.
* Missense mutations may retain function, depending on the chemistry of the new amino acid and its location in the protein. Nonsense mutations produce truncated and frequently nonfunctional proteins.
* A **frameshift mutation** results from an insertion or deletion of a number of nucleotides that is not a multiple of three. The change in reading frame alters every amino acid after the point of the mutation and results in a nonfunctional protein.
* **Spontaneous mutations** occur through DNA replication errors, whereas **induced mutations** occur through exposure to a **mutagen**.
* Mutagenic agents are frequently carcinogenic but not always. However, nearly all carcinogens are mutagenic.
* Chemical mutagens include base analogs and chemicals that modify existing bases. In both cases, mutations are introduced after several rounds of DNA replication.
* **Ionizing radiation,** such as X-rays and γ-rays, leads to breakage of the phosphodiester backbone of DNA and can also chemically modify bases to alter their base-pairing rules.
* **Nonionizing radiation** like ultraviolet light may introduce pyrimidine (thymine) dimers, which, during DNA replication and transcription, may introduce frameshift or point mutations.
* Cells have mechanisms to repair naturally occurring mutations. DNA polymerase has proofreading activity. Mismatch repair is a process to repair incorrectly incorporated bases after DNA replication has been completed.
* Pyrimidine dimers can also be repaired. In **nucleotide excision repair (dark repair)**, enzymes recognize the distortion introduced by the pyrimidine dimer and replace the damaged strand with the correct bases, using the undamaged DNA strand as a template. Bacteria and other organisms may also use **direct repair**, in which the photolyase enzyme, in the presence of visible light, breaks apart the pyrimidines.
* Through comparison of growth on the complete plate and lack of growth on media lacking specific nutrients, specific loss-of-function mutants called **auxotrophs** can be identified.
* The **Ames test** is an inexpensive method that uses auxotrophic bacteria to measure mutagenicity of a chemical compound. Mutagenicity is an indicator of carcinogenic potential.
{: data-bullet-style="bullet"}

### Multiple Choice

<div data-type="exercise">
<div data-type="problem" markdown="1">
Which of the following is a change in the sequence that leads to formation of a stop codon?

1.  missense mutation
2.  nonsense mutation
3.  silent mutation
4.  deletion mutation
{: data-number-style="upper-alpha"}

</div>
<div data-type="solution" markdown="1">
B

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
The formation of pyrimidine dimers results from which of the following?

1.  spontaneous errors by DNA polymerase
2.  exposure to gamma radiation
3.  exposure to ultraviolet radiation
4.  exposure to intercalating agents
{: data-number-style="upper-alpha"}

</div>
<div data-type="solution" markdown="1">
C

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Which of the following is an example of a frameshift mutation?

1.  a deletion of a codon
2.  missense mutation
3.  silent mutation
4.  deletion of one nucleotide
{: data-number-style="upper-alpha"}

</div>
<div data-type="solution" markdown="1">
D

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Which of the following is the type of DNA repair in which thymine dimers are directly broken down by the enzyme photolyase?

1.  direct repair
2.  nucleotide excision repair
3.  mismatch repair
4.  proofreading
{: data-number-style="upper-alpha"}

</div>
<div data-type="solution" markdown="1">
A

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Which of the following regarding the Ames test is true?

1.  It is used to identify newly formed auxotrophic mutants.
2.  It is used to identify mutants with restored biosynthetic activity.
3.  It is used to identify spontaneous mutants.
4.  It is used to identify mutants lacking photoreactivation activity.
{: data-number-style="upper-alpha"}

</div>
<div data-type="solution" markdown="1">
B

</div>
</div>

### Fill in the Blank

<div data-type="exercise">
<div data-type="problem" markdown="1">
A chemical mutagen that is structurally similar to a nucleotide but has different base-pairing rules is called a \_\_\_\_\_\_\_\_.

</div>
<div data-type="solution" markdown="1">
nucleoside analog

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
The enzyme used in light repair to split thymine dimers is called \_\_\_\_\_\_\_\_.

</div>
<div data-type="solution" markdown="1">
photolyase

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
The phenotype of an organism that is most commonly observed in nature is called the \_\_\_\_\_\_\_\_.

</div>
<div data-type="solution" markdown="1">
wild type

</div>
</div>

### True/False

<div data-type="exercise">
<div data-type="problem" markdown="1">
Carcinogens are typically mutagenic.

</div>
<div data-type="solution" markdown="1">
True

</div>
</div>

### Short Answer

<div data-type="exercise">
<div data-type="problem" markdown="1">
Why is it more likely that insertions or deletions will be more detrimental to a cell than point mutations?

</div>
</div>

### Critical Thinking

<div data-type="exercise">
<div data-type="problem" markdown="1">
Below are several DNA sequences that are mutated compared with the wild-type sequence: 3’-T A C T G A C T G A C G A T C-5’. Envision that each is a section of a DNA molecule that has separated in preparation for transcription, so you are only seeing the template strand. Construct the complementary DNA sequences (indicating 5’ and 3’ ends) for each mutated DNA sequence, then transcribe (indicating 5’ and 3’ ends) the template strands, and translate the mRNA molecules using the genetic code, recording the resulting amino acid sequence (indicating the N and C termini). What type of mutation is each?

<table summary="No Summary" class="unnumbered unstyled"><tbody>
<tr valign="top">
<td data-valign="top" data-align="left"><strong>Mutated DNA Template Strand #1: 3’-T A C T G T C T G A C G A T C-5’</strong><hr data-type="newline" />
Complementary DNA sequence:<hr data-type="newline" />
mRNA sequence transcribed from template:<hr data-type="newline" />
Amino acid sequence of peptide:<hr data-type="newline" />
Type of mutation:</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left"><strong>Mutated DNA Template Strand #2: 3’-T A C G G A C T G A C G A T C-5’</strong><hr data-type="newline" />
Complementary DNA sequence:<hr data-type="newline" />
mRNA sequence transcribed from template:<hr data-type="newline" />
Amino acid sequence of peptide:<hr data-type="newline" />
Type of mutation:</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left"><strong>Mutated DNA Template Strand #3: 3’-T A C T G A C T G A C T A T C-5</strong>’<hr data-type="newline" />
Complementary DNA sequence:<hr data-type="newline" />
mRNA sequence transcribed from template:<hr data-type="newline" />
Amino acid sequence of peptide:<hr data-type="newline" />
Type of mutation:</td>
</tr>
<tr valign="top">
<td data-valign="top" data-align="left"><strong>Mutated DNA Template Strand #4: 3’-T A C G A C T G A C T A T C-5’</strong><hr data-type="newline" />
Complementary DNA sequence:<hr data-type="newline" />
mRNA sequence transcribed from template:<hr data-type="newline" />
Amino acid sequence of peptide:<hr data-type="newline" />
Type of mutation:</td>
</tr>
</tbody></table>
</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Why do you think the Ames test is preferable to the use of animal models to screen chemical compounds for mutagenicity?

</div>
</div>

<div data-type="footnote-refs" markdown="1">
### Footnotes
{: data-type="footnote-refs-title"}

* {: data-type="footnote-ref" #footnote1} [1](#footnote-ref1){: data-type="footnote-ref-link"} <span data-type="footnote-ref-content">World Health Organization. “ Global Health Observatory (GHO) Data, HIV/AIDS.” http://www.who.int/gho/hiv/en/. Accessed August 5, 2016.</span>
* {: data-type="footnote-ref" #footnote2} [2](#footnote-ref2){: data-type="footnote-ref-link"} <span data-type="footnote-ref-content">World Health Organization. “ Global Health Observatory (GHO) Data, HIV/AIDS.” http://www.who.int/gho/hiv/en/. Accessed August 5, 2016.</span>
* {: data-type="footnote-ref" #footnote3} [3](#footnote-ref3){: data-type="footnote-ref-link"} <span data-type="footnote-ref-content">K.R. Tindall et al. “Changes in DNA Base Sequence Induced by Gamma-Ray Mutagenesis of Lambda Phage and Prophage.” *Genetics* 118 no. 4 (1988):551–560.</span>
{: data-list-type="bulleted" data-bullet-style="none"}

</div>

