---
title: "Whole Genome Methods and Pharmaceutical Applications of Genetic Engineering"
layout: page
---


### Learning Objectives

* Explain the uses of genome-wide comparative analyses
* Summarize the advantages of genetically engineered pharmaceutical products

Advances in molecular biology have led to the creation of entirely new fields of science. Among these are fields that study aspects of whole genomes, collectively referred to as whole-genome methods. In this section, we’ll provide a brief overview of the whole-genome fields of genomics, transcriptomics, and proteomics.

### Genomics, Transcriptomics, and Proteomics

The study and comparison of entire genomes, including the complete set of genes and their nucleotide sequence and organization, is called **genomics**{: data-type="term"}. This field has great potential for future medical advances through the study of the human genome as well as the genomes of infectious organisms. Analysis of microbial genomes has contributed to the development of new antibiotics, diagnostic tools, vaccines, medical treatments, and environmental cleanup techniques.

The field of **transcriptomics**{: data-type="term"} is the science of the entire collection of mRNA molecules produced by cells. Scientists compare gene expression patterns between infected and uninfected host cells, gaining important information about the cellular responses to infectious disease. Additionally, transcriptomics can be used to monitor the gene expression of **virulence factors**{: data-type="term" .no-emphasis} in microorganisms, aiding scientists in better understanding pathogenic processes from this viewpoint.

When genomics and transcriptomics are applied to entire microbial communities, we use the terms **metagenomics**{: data-type="term"} and **metatranscriptomics**{: data-type="term"}, respectively. Metagenomics and metatranscriptomics allow researchers to study genes and gene expression from a collection of multiple species, many of which may not be easily cultured or cultured at all in the laboratory. A DNA **microarray**{: data-type="term" .no-emphasis} (discussed in the previous section) can be used in metagenomics studies.

Another up-and-coming clinical application of genomics and transcriptomics is **pharmacogenomics**{: data-type="term"}, also called **toxicogenomics**{: data-type="term"}, which involves evaluating the effectiveness and safety of drugs on the basis of information from an individual’s genomic sequence. Genomic responses to drugs can be studied using experimental animals (such as laboratory rats or mice) or live cells in the laboratory before embarking on studies with humans. Changes in gene expression in the presence of a drug can sometimes be an early indicator of the potential for toxic effects. Personal genome sequence information may someday be used to prescribe medications that will be most effective and least toxic on the basis of the individual patient’s genotype.

The study of **proteomics**{: data-type="term"} is an extension of genomics that allows scientists to study the entire complement of proteins in an organism, called the **proteome**{: data-type="term" .no-emphasis}. Even though all cells of a multicellular organism have the same set of genes, cells in various tissues produce different sets of proteins. Thus, the genome is constant, but the proteome varies and is dynamic within an organism. Proteomics may be used to study which proteins are expressed under various conditions within a single cell type or to compare protein expression patterns between different organisms.

The most prominent disease being studied with proteomic approaches is cancer, but this area of study is also being applied to infectious diseases. Research is currently underway to examine the feasibility of using proteomic approaches to diagnose various types of hepatitis, tuberculosis, and HIV infection, which are rather difficult to diagnose using currently available techniques.<sup data-type="footnote-number" id="footnote-ref1">[1](#footnote1){: data-type="footnote-link"}</sup>

A recent and developing proteomic analysis relies on identifying proteins called **biomarkers**{: data-type="term"}, whose expression is affected by the disease process. Biomarkers are currently being used to detect various forms of cancer as well as infections caused by pathogens such as ***Yersinia pestis***{: data-type="term" .no-emphasis} and ***Vaccinia virus***{: data-type="term" .no-emphasis}.<sup data-type="footnote-number" id="footnote-ref2">[2](#footnote2){: data-type="footnote-link"}</sup>

Other “-omic” sciences related to genomics and proteomics include metabolomics, glycomics, and lipidomics, which focus on the complete set of small-molecule metabolites, sugars, and lipids, respectively, found within a cell. Through these various global approaches, scientists continue to collect, compile, and analyze large amounts of genetic information. This emerging field of **bioinformatics**{: data-type="term"} can be used, among many other applications, for clues to treating diseases and understanding the workings of cells.

Additionally, researchers can use **reverse genetics**{: data-type="term" .no-emphasis}, a technique related to classic **mutational analysis**{: data-type="term" .no-emphasis}, to determine the function of specific genes. Classic methods of studying gene function involved searching for the genes responsible for a given phenotype. Reverse genetics uses the opposite approach, starting with a specific DNA sequence and attempting to determine what phenotype it produces. Alternatively, scientists can attach known genes (called reporter genes) that encode easily observable characteristics to genes of interest, and the location of expression of such genes of interest can be easily monitored. This gives the researcher important information about what the gene product might be doing or where it is located in the organism. Common reporter genes include bacterial ***lacZ***{: data-type="term" .no-emphasis}, which encodes beta-galactosidase and whose activity can be monitored by changes in colony color in the presence of **X-gal**{: data-type="term" .no-emphasis} as previously described, and the gene encoding the jellyfish protein **green fluorescent protein**{: data-type="term" .no-emphasis} (GFP) whose activity can be visualized in colonies under ultraviolet light exposure ([\[link\]](#OSC_Microbio_12_03_GFP)).

 ![a) A photograph of mice with green fluorescent regions. B) A photograph of an agar plate with green fluorescent colonies. C) A photograph of blue and white colonies on an agar plate](../resources/OSC_Microbio_12_03_GFP.jpg "(a) The gene encoding green fluorescence protein is a commonly used reporter gene for monitoring gene expression patterns in organisms. Under ultraviolet light, GFP fluoresces. Here, two mice are expressing GFP, while the middle mouse is not. (b) GFP can be used as a reporter gene in bacteria as well. Here, a plate containing bacterial colonies expressing GFP is shown. (c) Blue-white screening in bacteria is accomplished through the use of the lacZ reporter gene, followed by plating of bacteria onto medium containing X-gal. Cleavage of X-gal by the LacZ enzyme results in the formation of blue colonies. (credit a: modification of work by Ingrid Moen, Charlotte Jevne, Jian Wang, Karl-Henning Kalland, Martha Chekenya, Lars A Akslen, Linda Sleire, Per &#xD8; Enger, Rolf K Reed, Anne M &#xD8;yan, Linda EB Stuhr; credit b: modification of work by &#x201C;2.5JIGEN.com&#x201D;/Flickr; credit c:  modification of work by American Society for Microbiology)"){: #OSC_Microbio_12_03_GFP}

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* How is genomics different from traditional genetics?
* If you wanted to study how two different cells in the body respond to an infection, what –omics field would you apply?
* What are the biomarkers uncovered in proteomics used for?
{: data-bullet-style="bullet"}

</div>

<div data-type="note" class="microbiology clinical-focus" markdown="1">
<div data-type="title">
Resolution
</div>
Because Kayla’s symptoms were persistent and serious enough to interfere with daily activities, Kayla’s physician decided to order some laboratory tests. The physician collected samples of Kayla’s blood, cerebrospinal fluid (CSF), and **synovial fluid**{: data-type="term" .no-emphasis} (from one of her swollen knees) and requested PCR analysis on all three samples. The PCR tests on the CSF and synovial fluid came back positive for the presence of ***Borrelia burgdorferi***{: data-type="term" .no-emphasis}, the bacterium that causes **Lyme disease**{: data-type="term" .no-emphasis}.

Kayla’s physician immediately prescribed a full course of the antibiotic **doxycycline**{: data-type="term" .no-emphasis}. Fortunately, Kayla recovered fully within a few weeks and did not suffer from the long-term symptoms of **post-treatment Lyme disease syndrome**{: data-type="term" .no-emphasis} (PTLDS), which affects 10–20% of Lyme disease patients. To prevent future infections, Kayla’s physician advised her to use insect repellant and wear protective clothing during her outdoor adventures. These measures can limit exposure to Lyme-bearing ticks, which are common in many regions of the United States during the warmer months of the year. Kayla was also advised to make a habit of examining herself for ticks after returning from outdoor activities, as prompt removal of a tick greatly reduces the chances of infection.

Lyme disease is often difficult to diagnose. *B. burgdorferi* is not easily cultured in the laboratory, and the initial symptoms can be very mild and resemble those of many other diseases. But left untreated, the symptoms can become quite severe and debilitating. In addition to two antibody tests, which were inconclusive in Kayla’s case, and the PCR test, a Southern blot could be used with *B. burgdorferi*-specific DNA probes to identify DNA from the pathogen. Sequencing of surface protein genes of *Borrelia* species is also being used to identify strains within the species that may be more readily transmitted to humans or cause more severe disease.

*Go back to the [previous](/m58849#fs-id1167742455813) Clinical Focus box.*

</div>

### Recombinant DNA Technology and Pharmaceutical Production

Genetic engineering has provided a way to create new pharmaceutical products called **recombinant DNA pharmaceuticals**{: data-type="term"}. Such products include antibiotic drugs, vaccines, and hormones used to treat various diseases. [\[link\]](#fs-id1167742466149) lists examples of recombinant DNA products and their uses.

For example, the naturally occurring antibiotic synthesis pathways of various ***Streptomyces***{: data-type="term" .no-emphasis} spp., long known for their antibiotic production capabilities, can be modified to improve yields or to create new antibiotics through the introduction of genes encoding additional enzymes. More than 200 new antibiotics have been generated through the targeted inactivation of genes and the novel combination of antibiotic synthesis genes in antibiotic-producing *Streptomyces* hosts.<sup data-type="footnote-number" id="footnote-ref3">[3](#footnote3){: data-type="footnote-link"}</sup>

Genetic engineering is also used to manufacture **subunit vaccines**{: data-type="term" .no-emphasis}, which are safer than other vaccines because they contain only a single antigenic molecule and lack any part of the genome of the pathogen (see [Vaccines](/m58888){: .target-chapter}). For example, a vaccine for hepatitis B is created by inserting a gene encoding a hepatitis B surface protein into a yeast; the yeast then produces this protein, which the human immune system recognizes as an antigen. The hepatitis B antigen is purified from yeast cultures and administered to patients as a vaccine. Even though the vaccine does not contain the hepatitis B virus, the presence of the antigenic protein stimulates the immune system to produce antibodies that will protect the patient against the virus in the event of exposure.<sup data-type="footnote-number" id="footnote-ref4">[4](#footnote4){: data-type="footnote-link"}</sup> <sup data-type="footnote-number" id="footnote-ref5">[5](#footnote5){: data-type="footnote-link"}</sup>

Genetic engineering has also been important in the production of other therapeutic proteins, such as **insulin**{: data-type="term" .no-emphasis}, **interferons**{: data-type="term" .no-emphasis}, and **human growth hormone**{: data-type="term" .no-emphasis}, to treat a variety of human medical conditions. For example, at one time, it was possible to treat diabetes only by giving patients pig insulin, which caused allergic reactions due to small differences between the proteins expressed in human and pig insulin. However, since 1978, recombinant DNA technology has been used to produce large-scale quantities of human insulin using *E. coli* in a relatively inexpensive process that yields a more consistently effective pharmaceutical product. Scientists have also genetically engineered *E. coli* capable of producing human growth hormone (HGH), which is used to treat growth disorders in children and certain other disorders in adults. The HGH gene was cloned from a cDNA library and inserted into *E. coli* cells by cloning it into a bacterial vector. Eventually, genetic engineering will be used to produce **DNA vaccines**{: data-type="term" .no-emphasis} and various gene therapies, as well as customized medicines for fighting cancer and other diseases.

<table summary="Some genetically engineered pharmaceuticals and applications. Atrial natriuretic peptide: Treatment of heart disease (e.g., congestive heart failure), kidney disease, high blood pressure. DNase: Treatment of viscous lung secretions in cystic fibrosis. Erythropoietin: Treatment of severe anemia with kidney damage. Factor VIII: Treatment of hemophilia. Hepatitis B vaccine: Prevention of hepatitis B infection. Human growth hormone: Treatment of growth hormone deficiency, Turner&#x2019;s syndrome, burns. Human insulin: Treatment of diabetes. Interferons: Treatment of multiple sclerosis, various cancers (e.g., melanoma), viral infections (e.g., Hepatitis B and C). Tetracenomycins: Used as antibiotics. Tissue plasminogen activator: Treatment of pulmonary embolism in ischemic stroke, myocardial infarction." class="span-all"><thead>
<tr valign="top">
<th colspan="2" data-valign="middle" data-align="center">Some Genetically Engineered Pharmaceutical Products and Applications</th>
</tr>
<tr valign="top">
<th data-valign="middle" data-align="center">Recombinant DNA Product</th>
<th data-valign="middle" data-align="center">Application</th>
</tr>
</thead><tbody>
<tr valign="top">
<td data-valign="middle" data-align="left">Atrial natriuretic peptide</td>
<td data-valign="middle" data-align="left">Treatment of heart disease (e.g., congestive heart failure), kidney disease, high blood pressure</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">DNase</td>
<td data-valign="middle" data-align="left">Treatment of viscous lung secretions in cystic fibrosis</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">Erythropoietin</td>
<td data-valign="middle" data-align="left">Treatment of severe anemia with kidney damage</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">Factor VIII</td>
<td data-valign="middle" data-align="left">Treatment of hemophilia</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">Hepatitis B vaccine</td>
<td data-valign="middle" data-align="left">Prevention of hepatitis B infection</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">Human growth hormone</td>
<td data-valign="middle" data-align="left">Treatment of growth hormone deficiency, Turner’s syndrome, burns</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">Human insulin</td>
<td data-valign="middle" data-align="left">Treatment of diabetes</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">Interferons</td>
<td data-valign="middle" data-align="left">Treatment of multiple sclerosis, various cancers (e.g., melanoma), viral infections (e.g., Hepatitis B and C)</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">Tetracenomycins</td>
<td data-valign="middle" data-align="left">Used as antibiotics</td>
</tr>
<tr valign="top">
<td data-valign="middle" data-align="left">Tissue plasminogen activator</td>
<td data-valign="middle" data-align="left">Treatment of pulmonary embolism in ischemic stroke, myocardial infarction</td>
</tr>
</tbody></table>

<div data-type="note" class="microbiology check-your-understanding" markdown="1">
* What bacterium has been genetically engineered to produce human insulin for the treatment of diabetes?
* Explain how microorganisms can be engineered to produce vaccines.
{: data-bullet-style="bullet"}

</div>

### RNA Interference Technology

In [Structure and Function of RNA](/m58837){: .target-chapter}, we described the function of mRNA, rRNA, and tRNA. In addition to these types of RNA, cells also produce several types of small noncoding RNA molecules that are involved in the regulation of gene expression. These include **antisense RNA**{: data-type="term"} molecules, which are complementary to regions of specific mRNA molecules found in both prokaryotes and eukaryotic cells. Non-coding RNA molecules play a major role in **RNA interference (RNAi)**{: data-type="term"}, a natural regulatory mechanism by which mRNA molecules are prevented from guiding the synthesis of proteins. RNA interference of specific genes results from the base pairing of short, single-stranded antisense RNA molecules to regions within complementary mRNA molecules, preventing protein synthesis. Cells use RNA interference to protect themselves from viral invasion, which may introduce double-stranded RNA molecules as part of the viral replication process ([\[link\]](#OSC_Microbio_12_03_Noncoding)).

 ![A eukaryotic cell transcribes a region of DNA into mrNA. Antisense mRNA then binds to the this mRNA to produce a double stranded region. This region is not translated (which means that ribosomes do not bind to the mRNA to produce proteins).](../resources/OSC_Microbio_12_03_Noncoding.jpg "Cells like the eukaryotic cell shown in this diagram commonly make small antisense RNA molecules with sequences complementary to specific mRNA molecules. When an antisense RNA molecule is bound to an mRNA molecule, the mRNA can no longer be used to direct protein synthesis. (credit: modification of work by Robinson R)"){: #OSC_Microbio_12_03_Noncoding}

Researchers are currently developing techniques to mimic the natural process of RNA interference as a way to treat viral infections in eukaryotic cells. RNA interference technology involves using small interfering RNAs (siRNAs) or microRNAs (miRNAs) ([\[link\]](#OSC_Microbio_12_03_siRNA)). siRNAs are completely complementary to the mRNA transcript of a specific gene of interest while miRNAs are mostly complementary. These double-stranded RNAs are bound to DICER, an endonuclease that cleaves the RNA into short molecules (approximately 20 nucleotides long). The RNAs are then bound to RNA-induced silencing complex (RISC), a ribonucleoprotein. The siRNA-RISC complex binds to mRNA and cleaves it. For miRNA, only one of the two strands binds to RISC. The miRNA-RISC complex then binds to mRNA, inhibiting translation. If the miRNA is completely complementary to the target gene, then the mRNA can be cleaved. Taken together, these mechanisms are known as **gene silencing**{: data-type="term"}.

 ![Double stranded RNA can be produced from DNA in the nucleus. Dicer than cuts this dsRNA into either miRNA or siRNA. miRNA is an imperfect match and only one strand is usually incorporated into RISC. This blocks translation but the mRNA is stable. The RISC is stuck on the target. The siRNA has a perfect match and is incorporated into RISC. This triggers mRNA cleavage.](../resources/OSC_Microbio_12_03_siRNA.jpg "This diagram illustrates the process of using siRNA or miRNA in a eukaryotic cell to silence genes involved in the pathogenesis of various diseases. (credit: modification of work by National Center for Biotechnology Information)"){: #OSC_Microbio_12_03_siRNA}

### Key Concepts and Summary

* The science of **genomics** allows researchers to study organisms on a holistic level and has many applications of medical relevance.
* **Transcriptomics** and **proteomics** allow researchers to compare gene expression patterns between different cells and shows great promise in better understanding global responses to various conditions.
* The various –omics technologies complement each other and together provide a more complete picture of an organism’s or microbial community’s (**metagenomics**) state.
* The analysis required for large data sets produced through genomics, transcriptomics, and **proteomics** has led to the emergence of **bioinformatics**.
* **Reporter genes** encoding easily observable characteristics are commonly used to track gene expression patterns of genes of unknown function.
* The use of recombinant DNA technology has revolutionized the pharmaceutical industry, allowing for the rapid production of high-quality **recombinant DNA pharmaceuticals** used to treat a wide variety of human conditions.
* **RNA interference** technology has great promise as a method of treating viral infections by silencing the expression of specific genes
{: data-bullet-style="bullet"}

### Multiple Choice

<div data-type="exercise">
<div data-type="problem" markdown="1">
The science of studying the entire collection of mRNA molecules produced by cells, allowing scientists to monitor differences in gene expression patterns between cells, is called:

1.  genomics
2.  transcriptomics
3.  proteomics
4.  pharmacogenomics
{: type="a"}

</div>
<div data-type="solution" markdown="1">
B

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
The science of studying genomic fragments from microbial communities, allowing researchers to study genes from a collection of multiple species, is called:

1.  pharmacogenomics
2.  transcriptomics
3.  metagenomics
4.  proteomics
{: type="a"}

</div>
<div data-type="solution" markdown="1">
C

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
The insulin produced by recombinant DNA technology is

1.  a combination of *E. coli* and human insulin.
2.  identical to human insulin produced in the pancreas.
3.  cheaper but less effective than pig insulin for treating diabetes.
4.  engineered to be more effective than human insulin.
{: type="a"}

</div>
<div data-type="solution" markdown="1">
B

</div>
</div>

### Fill in the Blank

<div data-type="exercise">
<div data-type="problem" markdown="1">
The application of genomics to evaluate the effectiveness and safety of drugs on the basis of information from an individual’s genomic sequence is called \_\_\_\_\_\_\_\_\_\_\_\_.

</div>
<div data-type="solution" markdown="1">
pharmacogenomics or toxicogenomics

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
A gene whose expression can be easily visualized and monitored is called a \_\_\_\_\_\_\_\_.

</div>
<div data-type="solution" markdown="1">
reporter gene

</div>
</div>

### True/False

<div data-type="exercise">
<div data-type="problem" markdown="1">
RNA interference does not influence the sequence of genomic DNA.

</div>
<div data-type="solution" markdown="1">
true

</div>
</div>

### Short answer

<div data-type="exercise">
<div data-type="problem" markdown="1">
If all cellular proteins are encoded by the cell’s genes, what information does proteomics provide that genomics cannot?

</div>
</div>

### Critical Thinking

<div data-type="exercise">
<div data-type="problem" markdown="1">
What are some advantages of cloning human genes into bacteria to treat human diseases caused by specific protein deficiencies?

</div>
</div>

<div data-type="footnote-refs" markdown="1">
### Footnotes
{: data-type="footnote-refs-title"}

* {: data-type="footnote-ref" #footnote1} [1](#footnote-ref1){: data-type="footnote-ref-link"} <span data-type="footnote-ref-content">E.O. List, D.E. Berryman, B. Bower, L. Sackmann-Sala, E. Gosney, J. Ding, S. Okada, and J.J. Kopchick. “The Use of Proteomics to Study Infectious Diseases.” *Infectious Disorders-Drug Targets* (Formerly *Current Drug Targets-Infectious Disorders*) *8* no. 1 (2008): 31–45.</span>
* {: data-type="footnote-ref" #footnote2} [2](#footnote-ref2){: data-type="footnote-ref-link"} <span data-type="footnote-ref-content">Mohan Natesan, and Robert G. Ulrich. “Protein Microarrays and Biomarkers of Infectious Disease.” *International Journal of Molecular Sciences 11* no. 12 (2010): 5165–5183.</span>
* {: data-type="footnote-ref" #footnote3} [3](#footnote-ref3){: data-type="footnote-ref-link"} <span data-type="footnote-ref-content">Jose-Luis Adrio and Arnold L. Demain. “Recombinant Organisms for Production of Industrial Products.” *Bioengineered Bugs 1* no. 2 (2010): 116–131.</span>
* {: data-type="footnote-ref" #footnote4} [4](#footnote-ref4){: data-type="footnote-ref-link"} <span data-type="footnote-ref-content">U.S. Department of Health and Human Services. “Types of Vaccines.” 2013. http://www.vaccines.gov/more\\\_info/types/#subunit. Accessed May 27, 2016.</span>
* {: data-type="footnote-ref" #footnote5} [5](#footnote-ref5){: data-type="footnote-ref-link"} <span data-type="footnote-ref-content">The Internet Drug List. *Recombivax*. 2015. http://www.rxlist.com/recombivax-drug.htm. Accessed May 27, 2016.</span>
{: data-list-type="bulleted" data-bullet-style="none"}

</div>

